enclave/src/main/java/org/hyperledger/besu/enclave/EnclaveFactory.java

```diff
@@ -20,10 +20,12 @@
 import java.net.URI;
 import java.nio.charset.StandardCharsets;
 import java.nio.file.AccessDeniedException;
+import java.nio.file.Files;
```

Comment:
These changes are unrelated to tuweni, move them to a separate PR please.

Reply:
It seems this upgrade of Tuweni changes dependencies, especially removing Guava. So we have to add Guava to the dependencies, or have to use the native Files class. I chose to go down that way as it was the only reference to Guava in that particular subproject. Would you rather I change to add Guava explicitly in dependencies?

Reply:
I would rather the dependency be changed.

Reply:
OK fixed.